FASTN.ftd

```diff
@@ -691,7 +691,8 @@ skip: true
   document: backend/ftd-redirect.ftd
 - Using `fastn` With Django Or Other Backends: /django/
   document: backend/django.ftd
-- Enviroment Variables: /backend/env-vars/
+- Enviroment Variables: /env/
+  document: backend/env-vars
```

Comment:
Enforcing file extensions is the right move, I'll make an issue about this in [fastn-stack/fastn](https://github.com/fastn-stack/fastn/). I have [added the file extension ](https://github.com/fastn-stack/fastn.com/pull/509/commits/fbfc977d8f64524c531d13640941007ab919f865) so that we can move forward with this PR.

Reply:
https://github.com/fastn-stack/fastn/issues/1444